kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/BaseOperation.java

```diff
@@ -1009,4 +1003,24 @@ public static URL appendListOptionParams(URL base, ListOptions listOptions) {
     return urlBuilder.build();
   }
 
+  @Override
+  public List<T> deleteList(List<T> items) throws ListDeleteException {
```

Comment:
There is some redundancy between the `delete(List)` and the `deleteList(List)` here. Maybe, we could align these two in a single private method?
In addition to that, `delete(List)` does a `null` check on the parameter. `deleteList(List)` doesn't.